qiime/sdk/tests/test_method.py

```diff
@@ -40,6 +40,9 @@ def setUp(self):
                 'int1': (qiime.plugin.Int, int),
                 'int2': (qiime.plugin.Int, int)
             },
+            defaults={
+                'int1': 42,
```

Comment:
Comparing signatures for equality ignores `defaults`. Can you add a comparison of `defaults` in `PipelineSignature.__eq__`? These tests will then start failing (and the visualizer tests below) since `'int1'` doesn't have a default value in the registered function (same issue for `'key_label'` below). You can set `defaults={}` for both cases to fix the tests.